<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFileDialog</class>
 <widget class="QDialog" name="LoadFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Data File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Data File</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineDataFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnDataFile">
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Apply preset</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboPreset"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Field Separator</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboFieldSeparator"/>
       </item>
       <item row="6" column="1">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineCustomFieldSeparator">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Decimal separator</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboDecimalSeparator"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Thousands separator</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="comboGroupSeparator"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Comment string</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="lineComment"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Column</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="spinColumn">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="checkLabelRow">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSpinBox" name="spinLabelRow">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data Row</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="spinDataRow">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_10">
         <property name="toolTip">
          <string>Enables auto reload</string>
         </property>
         <property name="text">
          <string>Dynamic session</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QCheckBox" name="checkDynamicSession">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Label Row</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QCheckBox" name="checkTimeInMilliSeconds">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Label Row Present?</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Time data in milliseconds</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Correct STMStudio bad reads</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QCheckBox" name="checkStmStudioCorrection">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tablePreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineDataFile</tabstop>
  <tabstop>btnDataFile</tabstop>
  <tabstop>comboPreset</tabstop>
  <tabstop>comboFieldSeparator</tabstop>
  <tabstop>lineCustomFieldSeparator</tabstop>
  <tabstop>comboDecimalSeparator</tabstop>
  <tabstop>comboGroupSeparator</tabstop>
  <tabstop>lineComment</tabstop>
  <tabstop>spinColumn</tabstop>
  <tabstop>checkLabelRow</tabstop>
  <tabstop>spinLabelRow</tabstop>
  <tabstop>spinDataRow</tabstop>
  <tabstop>checkDynamicSession</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
